tests/parser/types/numbers/test_num256.py

```diff
@@ -52,3 +52,15 @@ def _num256_le(x: num256, y: num256) -> bool:
     assert c._num256_lt(y, x) is True
 
     print("Passed num256 operation tests")
+
+def test_num256_with_exponents():
+    exp_code = """
+def _num256_exp(x: num256, y: num256) -> num256:
+        return num256_exp(x,y)
+    """
+
+    c = get_contract(exp_code)
+    assert c._num256_exp(2, 3) == 8
+    assert c._num256_exp(2**128, 2) == 0
+    assert c._num256_exp(2**64, 2) == 2**128
+    assert c._num256_exp(7**23, 3) == 20500514515695490612229010908095867391439626248463723805607
```

Comment:
Same as above, you could write `assert c._num256_exp(7**23, 3) == (7**23)**3` showing python == viper